datamol/__init__.py

```diff
@@ -127,3 +128,8 @@
 from .graph import get_all_path_between
 from .graph import match_molecular_graphs
 from .graph import reorder_mol_from_template
+
+from .reactions import rxn_from_smarts
+from .reactions import rxn_from_block
+from .reactions import apply_reaction
+from .reactions import is_reaction_ok
```

Comment:
@zhu0619 I think we should decode whether we want all the reaction functions available to `dm.*` or with `dm.reactions.*` (but not a mix of both)? I think I prefer `dm.reactions.*`. Do you have a preference?